victor-abz/free-mentors
# Ch incorporate feedback #168247159

#### What does this PR do?
Make the site responsive
Create a Good ReadMe
Create Pull Request Template in .git folder

#### Description of Task to be completed
**Make the site responsive:** Elements were being not accessible on different levels of zoom. I completed it by creating Different media queries and tested the responsiveness with google chrome developers tool(Inspect **ctrl+shift+i / cmd+shift+i**)

**Create a Good ReadMe:** The read has included some screenshot of the UI. removed unimportant dependencies list.
Create Pull Request Template in .git folder

#### How should this be manually tested?
This can be tested by downloading the UI and check the HTML files responsiveness and open google chrome and press **ctrl+shift+i / cmd+shift+i**. and check the responsiveness by clicking the switch to devices in inspecting.
Please refer to _https://www.atlantic.net/hipaa-compliant-wordpress-hosting/how-to-test-responsive-design-device-mode-chrome-developer-tools/_
for how to check with google developers' tools.

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
[#168247159]

#### Screenshots
![4](https://user-images.githubusercontent.com/52409462/64171355-bd4b9c80-ce52-11e9-8af6-11c8f0fea8a4.PNG)
![2](https://user-images.githubusercontent.com/52409462/64171384-d18f9980-ce52-11e9-9e62-0b715c3a5f53.PNG)
![log1](https://user-images.githubusercontent.com/52409462/64171457-03a0fb80-ce53-11e9-959f-29b21047ae1d.PNG)
![log2](https://user-images.githubusercontent.com/52409462/64171463-07cd1900-ce53-11e9-8bf8-40ee0d44aa4a.PNG)
![1](https://user-images.githubusercontent.com/52409462/64171338-b45acb00-ce52-11e9-85b0-3630513cda24.PNG)
